README.md

```diff
@@ -53,30 +53,30 @@ Then rerun bundler and everything will work. This is being tracked over in https
 Beyond that, the setup instructions are now the same for Intel or M1 macs, as you can install homebrew normally and go from there.
 
 #### Docker Compose
+Dev environment setup using docker and docker-compose
 
-For quick development and testing, the server can be run locally using Docker Compose and the command:
+##### Prerequisite 
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker compose](https://docs.docker.com/compose/install/)
```

Comment:
[nit] Can we say something like "Install docker from <>" so it reads more like a user manual.